src/sentry/static/sentry/app/views/settings/components/forms/fieldFromConfig.jsx

```diff
@@ -83,14 +64,15 @@ export default class FieldFromConfig extends React.Component {
         return <TextareaField {...props} />;
       case 'choice':
       case 'select':
+      case 'array':
```

Comment:
is there a reason for needing 3 aliases for this?

Reply:
Potential backwards compatibility.